packages/web/src/scss/components/Footer/index.html

```diff
@@ -0,0 +1,480 @@
+{{#> web/layout/plain }}
+
+<section class="UNSTABLE_Section">
+  <h2 class="docs-Heading">Basic Usage</h2>
+  <div class="docs-Stack">
+
+    <footer class="bg-cover pt-1100 pb-1000">
+      <div class="Container">
+
+        <!-- Grid with navigation links -->
+        <div
+          class="Grid Grid--cols-1 Grid--tablet--cols-2 Grid--desktop--cols-4"
+          style="--grid-spacing-x: var(--spirit-space-800); --grid-spacing-y: var(--spirit-space-800)"
+        >
+
+          <!-- Navigation links 1 -->
+          <nav aria-labelledby="footer-part-one">
```

Comment:
🙃

Reply:
I think this is OK:

![image](https://github.com/user-attachments/assets/c43684dd-b089-4a7f-8d33-d9176663cd2e)

Reply:
yes, I was just pointing again to the `footer-part-one` id :) but you already solved this.